src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -1,113 +1,98 @@
-"item.mmorpg.mat.infused_iron": "下品精华",
-"item.mmorpg.mat.crystallized_essence": "上品精华",
-"item.mmorpg.mat.golden_orb": "中品精华",
-"item.mmorpg.mat.mythic_essence": "精品精华",
-"item.mmorpg.awaken_runeword": "觉醒符文之语",
-"item.mmorpg.identify_tome": "鉴定卷册",
-"item.mmorpg.teleport_scroll": "回城卷轴",
-"item.mmorpg.prefix": "前缀",
-"item.mmorpg.suffix": "后缀",
-	"block.mmorpg.scrabble": "拼字宝箱",
-	"block.mmorpg.map_chest": "地图宝箱",
-"_comment": "SPELL TOOLTIPS",
-"mmorpg.tooltip.aoe_spell_explosion": "释放元素弹体，命中后造成范围伤害.",
-"mmorpg.tooltip.single_target_spell_bolt": "释放元素弹体，对单个目标造成伤害.",
-"mmorpg.tooltip.spell_instant_heal": "立即治疗施法者.",
-"mmorpg.tooltip.aoe_bomb_spell": "留下一枚炸弹，之后造成范围爆炸伤害.",
-"mmorpg.tooltip.spell_self_regen": "缓慢的持续治疗施法者的生命值.",
-"mmorpg.tooltip.aoe_spell_nova": "伤害你周围的所有目标.",
-"mmorpg.tooltip.ele_bonus_atk_dmg_spell": "给予增益效果, 基于法力造成额外伤害.",
-"_comment": "REPAIR CAPACITORS",
-"item.mmorpg.capacitor0": "凡品能效珠",
-"item.mmorpg.capacitor1": "下品能效珠",
-"item.mmorpg.capacitor2": "中品能效珠",
-"item.mmorpg.capacitor3": "上品能效珠",
-"item.mmorpg.capacitor4": "精品能效珠",
-"item.mmorpg.capacitor5": "珍品能效珠",
-"_comment": "ITEM TOOLTIPS",
-"death.attack.mmorpg.custom_damage": "%1$s 被残忍地杀害了",
-"_comment": "MAPS",
-"mmorpg.name.map": "冒险地图",
-"mmorpg.tooltip.put_in_mapdevice": "右键点击地图装置使用",
-"mmorpg.tooltip.press_shift_more_info": "[Shift]: 信息, [Alt]: 描述, [Ctrl]: 附魔",
-"mmorpg.tooltip.affix_rarity_lootbonus": "词缀品级战利品奖励",
-"curios.identifier.salvage_bag": "回收袋",
-"curios.identifier.bracelet": "手镯",
-"entity.mmorpg.trader.none": "地牢商人",
-"block.mmorpg.dungeon_portal": "地牢传送门",
-
-"_comment": " ENTITIES",
-"entity.mmorpg.spirit_wolf_pet": "猎狼（召唤物）",
-"entity.mmorpg.zombie_pet": "僵尸（召唤物）",
-"entity.mmorpg.skeleton_pet": "骷髅（召唤物）",
-
-"_comment": " [CATEGORY]: MISC - NAMES",
-
-	"mmorpg.mob_affix.accurate": "精准",
-	"mmorpg.mob_affix.cold": "寒冷",
-	"mmorpg.mob_affix.evasive": "闪避",
-	"mmorpg.mob_affix.flaming": "燃烧",
-	"mmorpg.mob_affix.lightning": "雷电",
-	"mmorpg.mob_affix.of_conductivity": "导电",
-	"mmorpg.mob_affix.of_harsh_winter": "严冬",
-	"mmorpg.mob_affix.of_leeching": "吸血",
-	"mmorpg.mob_affix.of_liquid_death": "死亡之水",
-	"mmorpg.mob_affix.of_mountains": "山脉",
-	"mmorpg.mob_affix.of_raging_volcano": "火山爆发",
-	"mmorpg.mob_affix.of_retribution": "报应",
-	"mmorpg.mob_affix.of_rock_skin": "坚硬皮肤",
-	"mmorpg.mob_affix.of_scalding": "灼热",
-	"mmorpg.mob_affix.of_the_cobra": "眼镜蛇",
-	"mmorpg.mob_affix.of_the_sniper": "狙击手",
-	"mmorpg.mob_affix.of_thunder_strikes": "雷电打击",
-	"mmorpg.mob_affix.of_toxic_fumes": "毒烟",
-	"mmorpg.mob_affix.of_venom_shock": "毒液冲击",
-	"mmorpg.mob_affix.resistant": "抗性",
-	"mmorpg.mob_affix.venom": "毒牙",
+	"block.mmorpg.egg_loot_crate": "战利品奖励箱",
+	"block.mmorpg.ore_block0": "凡品矿石",
+	"block.mmorpg.ore_block1": "下品矿石",
+	"block.mmorpg.ore_block2": "中品矿石",
+	"block.mmorpg.ore_block3": "上品矿石",
+	"block.mmorpg.ore_block4": "精品矿石",
+	"block.mmorpg.ore_block5": "玄品矿石",
+	"itemGroup.mmorpg_main": "挖矿与砍杀",
+	"itemGroup.mmorpg_uniques": "珍品装备",
```

Comment:
Honesly, this translation is not right. The most proper translation, also used in the RPG game like DOS2, is "独特", means literally "Unique". But I still can't change this because of the user's habit.